src/main/java/slimeknights/tconstruct/library/tools/nbt/LazyToolStack.java

```diff
@@ -34,6 +34,7 @@ public static LazyToolStack from(ToolStack tool) {
   }
 
   /** Gets the tool for this instance */
+  @NotNull
```

Comment:
I annotate methods as `Nonnull` at  a package level, you shouldn't need to set it on individual methods.

Reply:
Ah, for some reason Intellij didn't seem to recognize that. I'll undo that tomorrow

Reply:
You have to tell IntelliJ that I like javax annotations, just search `Nonnull` in settings, should be under Inspection > Configure Annotations. There is also a chance the package is missing an annotation (as it has to be added to every single package manually)

Reply:
The package is annotated correctly